components/dashboard/src/Login.tsx

```diff
@@ -34,7 +35,13 @@ function Item(props: { icon: string; iconSize?: string; text: string }) {
 }
 
 export function markLoggedIn() {
-    document.cookie = GitpodCookie.generateCookie(window.location.hostname);
+    document.cookie = GitpodCookie.generateCookie(GitpodCookie.NAME, GitpodCookie.VALUE, window.location.hostname);
+    // setting document.cookie more than once doesn't replace, only appends
+    document.cookie = GitpodCookie.generateCookie(
```

Comment:
I don't understand why we need to set it here. :thinking:

Reply:
I realize I overthought this and you're right! We only need to set all the consent cookies once there is an explicit choice made by the user. Will make the change. Thank you!